homeassistant/components/xiaomi_miio/number.py

```diff
@@ -127,6 +128,16 @@ class OscillationAngleValues:
         method="async_set_favorite_level",
         entity_category=EntityCategory.CONFIG,
     ),
+    FEATURE_SET_FAVORITE_LEVEL_MIOT: XiaomiMiioNumberDescription(
```

Comment:
Maybe we should instead replace the max value for this model type as we do for `OSCILLATION_ANGLE_VALUES`?

Reply:
ok, i will try like that.

Reply:
done